site/docs/guide/filter-queries.md

```diff
@@ -347,8 +347,24 @@ In this case, you will be provided with helpful error messages.
 The validation of the filter queries happens only once, when the bot is initialized and `bot.on()` is called.
 
 On start-up, grammY derives a predicate function from the filter query by splitting it into its query parts.
-Every part will be mapped to a function that performs a single `in` check, or two checks if the part is omitted and two values need to be checked.
+Every part will be mapped to a function that performs a single truthiness check for an object property, or two checks if the part is omitted and two values need to be checked.
```

Comment:
@KnorpelSenf Could you provide an example for the scenario mentioned in this line `two checks if the part is omitted and two values need to be checked`? I'm a bit confused about when the part is omitted.

Reply:
`":text"` will be expanded to `["message:text", "channel_post:text"]` so the omitted value before the colon requires two properties to be checked.

Reply:
Right. That makes sense now. Thanks.